deployment/ansible/roles/model-my-watershed.docker/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: Install Docker Compose
-  pip:
-    name: docker-compose
-    version: "{{ docker_compose_version }}"
+  pip: name="{{ item.name }}" version={{ item.version }} state=present executable=pip
+  with_items:
+    - { name: "docker", version: "{{ docker_python_version }}" }
```

Comment:
Can you describe why we are installing the `docker` Python package here?

Reply:
Good catch, this was required at some odd point in between, but is no longer required. Removed in 2e8a492.